ytdlbot/sp_downloader.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/local/bin/python3
+# coding: utf-8
+
+# ytdlbot - sp_downloader.py
+# 3/16/24 16:32
+#
+
+__author__ = "Benny <benny.think@gmail.com>, SanujaNS <sanujas@sanuja.biz>"
```

Comment:
You can just leave yourself there:-) I wouldn't mind

Reply:
🙇‍♂️